migration/src/download_jira.py

```diff
@@ -41,9 +41,16 @@ def download_issue(num: int, dump_dir: Path) -> bool:
     if res.status_code != 200:
         logger.warning(f"Can't download {issue_id}. status code={res.status_code}, message={res.text}")
         return False
+    data = res.json()
+    if "key" not in data:
+        logger.warning(f"The issue's key does not exist. Skipped {issue_id}")
```

Comment:
Sheesh let's hope this does not ever happen.

Reply:
Yes this shouldn't happen. just for safety...